inlong-sort/sort-connectors/doris/README.md

```diff
@@ -1,49 +0,0 @@
-# Overview
```

Comment:
I'm sorry, but why remove this file?

Reply:
Because the doris community recently provided the flink-doris-connector-1.13_2.11 version to the maven public repository, I think there is no need to note how to compile the 2.11 version.